ui/css/module/_footnotes.scss

```diff
@@ -5,14 +5,38 @@
   padding: 2px 4px;
   font-size: 0.6em;
   margin-left: .3em;
+  position: relative;
 
   &.active {
-    border-color: $color-primary-darker;
-    background: $color-primary-darker;
-    color: $color-white;
+    @extend .active-footnote-style;
+
+    &::before {
+      @extend .rotated-triangle;
+      z-index: 10;
+      content: '';
+      position: absolute;
+    }
   }
 }
 
+.active-footnote-style {
+  border-color: $color-primary-darker;
+  background: $color-primary-darker;
+  color: $color-white;
+}
+
+.rotated-triangle {
+  top: 39px;
+  background: $color-white;
+  height: 18px;
+  width: 18px;
+  left: 14px;
```

Comment:
Is there a way to do this (in CSS) based on a percentage rather than a fixed offset? With the potential for different fonts we might have very different "middle"s for a footnote citation (e.g. "Footnote 88" is much wider than "Footnote 1")